build/patches/Experimental-user-scripts-support.patch

```diff
@@ -2938,6 +2948,49 @@ new file mode 100755
 +  return true;
 +}
 +
++const std::u16string FileErrorString(base::File::Error error) {
```

Comment:
There is: https://source.chromium.org/chromium/chromium/src/+/main:base/files/file.cc;drc=5630be48e70e6de3514eb1f1e44c235bbec41d09;bpv=1;bpt=1;l=125

`ErrorToString`: why not use that one instead?

Reply:
because I need it to be talking and above all `u16string`

Reply:
Wouldn't it be better to convert the result of a call to `ErrorToString` to `u16string` then?

Reply:
There is this: https://source.chromium.org/chromium/chromium/src/+/main:base/strings/utf_string_conversions.h;l=51;bpv=1;bpt=1?q=base::UTF8ToUTF16&ss=chromium%2Fchromium%2Fsrc

`ASCIIToUTF16` should work.

Reply:
thanks perfect.

Reply:
fixed in https://github.com/bromite/bromite/commit/8b9bc0f0710adaf3da7520c9af73a12ccef23a45